src/components/Checkbox/styles/CheckboxLabel.js

```diff
@@ -5,12 +5,14 @@ import icons from 'components/Icon/icons';
 
 const SIZE = '30px';
 const CHECK_SIZE = '21px';
+const SMALL_SIZE = '20px';
+const SMALL_CHECK_SIZE = '1em';
 
-export default styled.label`
+const getCheckboxLabel = (size, checkSize) => styled.label`
```

Comment:
Use `styled(CheckboxLabel)` instead, so that they will have the same base classname.